apis/interface/servo/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+cargo-features = ["edition2024"]
```

Comment:
Why do we need this?

Reply:
@inesmaria08 ?

Reply:
It wouldn't compile before, but I believe it's working now.